program/build.rs

```diff
@@ -93,6 +93,18 @@ impl Runtime {
     }
 }
 
+/// Get profile name.
+fn get_build_profile_name() -> String {
+    // The profile name is always the 3rd last part of the path (with 1 based indexing).
+    // e.g. /code/core/target/cli/build/my-build-info-9f91ba6f99d7a061/out
```

Comment:
Where is the profile tho?
```suggestion
    // e.g. /code/core/target/cli/build/my-build-info-9f91ba6f99d7a061/out
```

Reply:
Made it more clear